run_app.sh

```diff
@@ -1,25 +1,40 @@
-TEMPDIR=`mktemp -d`
+# VALIDATES IF WE HAVE CLOWDER ENV 
+if [ -z "$ACG_CONFIG"]; then
```

Comment:
NIT: Personally, I like to have the "true" condition  on the if

Reply:
I will keep this in mind! I don't make so much changes for now so we can move with the tests!!

Thanks for the advice